app/script/backup/BackupRepository.js

```diff
@@ -78,12 +78,6 @@ z.backup.BackupRepository = class BackupRepository {
           throw new z.backup.CancelError();
         }
         progressCallback(rows.length);
-
-        const isEventTable = tableName === z.storage.StorageSchemata.OBJECT_STORE.EVENTS;
```

Comment:
Wow, this is not normal at all. This is something we added this morning, it's not supposed to be removed

Reply:
There must have been a weird git operation I think

Reply:
weird ... i will add it back in